src/platform/packages/private/kbn-esql-editor/src/esql_editor.tsx

```diff
@@ -87,7 +100,7 @@ const triggerControl = async (
   });
 };
 
-export const ESQLEditor = memo(function ESQLEditor({
+const ESQLEditorInternal = memo(function ESQLEditor({
```

Comment:
```suggestion
const ESQLEditorInternal = function ESQLEditor({
```
Nit: it's double memoized now that we're using `withRestorableState` which already memoizes.